# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1344180248 -3600
#      Sun Aug 05 16:24:08 2012 +0100
# Branch gui
# Node ID acb7c0e6c317865f083bef2f92c014287ab13c51
# Parent  0ba814cc489a113eedbe4d321469a01526cec13e
Force update of .hgsubstate

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,2 +1,2 @@
 0e3af50c9e20938bd1cea0182bf749ce61cb6782 gnulib
-f91e00f96bc29f7f6395f08c828e09599f945511 gui/qterminal
+0f7cc1d7ef5c34b146ff71207ada3ec87a24097c gui/qterminal
